assets/js/load_more.js

```diff
@@ -0,0 +1,36 @@
+jQuery(document).ready(function () {
+    initLoadMore();
+})
+
+function initLoadMore() {
+    jQuery('.jtf-load-more').on('click', function (e) {
+        e.preventDefault();
+        var $loadmore = jQuery(this),
+            link = $loadmore.attr('href'),
+            selector = $loadmore.attr('data-selector'),
+            container = $loadmore.attr('data-container'),
+            attribute = $loadmore.attr('data-attribute'),
+            cont = ( (attribute == 'class' ) ? '.' : '#') + selector;
```

Comment:
inside selector you should write the selector with '.' or '#'. no need to do this with separate parameter.
you can track this on backend just getting first char